### PR TITLE
Feature/866

### PR DESCRIPTION
**Pull request by medullaskyline**
#866, #914, #908, #918

Can't test patient_details and sample_details locally since required tables don't exist in vagrant db.

### CODE REVIEW

File: api/Cohort.py [hunk 1]
```diff
@@ -273,11 +275,13 @@ def cohorts_list(self, request):
                 cursor.execute(query_str, query_tuple)
                 data = []
                 for row in cursor.fetchall():
-                    filter_query_str = 'select name, value ' \
-                                       'from cohorts_filters ' \
-                                       'where cohorts_filters.resulting_cohort_id=%s'
+                    print >> sys.stderr, '\n\n' + str(row['id'])
```
Comment (phyllers) [inline api/Cohort.py]: Can we remove this?

File: vagrant@ubuntu-14 [hunk 2]
```diff
@@ -0,0 +1,81 @@
+"""
```
Comment (phyllers) [inline vagrant@ubuntu-14]: What is this file for?

Comment (medullaskyline) [inline vagrant@ubuntu-14]: That's weird. This is `isb_auth.py`.

Comment (medullaskyline) [inline vagrant@ubuntu-14]: How do I delete it?

Comment (phyllers) [inline vagrant@ubuntu-14]: git rm?

File: app.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@
 #
 ###
 
-runtime: custom
+runtime: python27
```
Comment (phyllers) [inline app.yaml]: sorry!

Comment (medullaskyline) [inline app.yaml]: thought i got that one :-/

**Pull request merged**